types/mongoose/index.d.ts

```diff
@@ -2697,8 +2648,8 @@ declare module "mongoose" {
      * @param options.validateBeforeSave set to false to save without validating.
      * @param fn optional callback
      */
-    save(options?: SaveOptions, fn?: (err: any, product: this, numAffected: number) => void): Promise<this>;
-    save(fn?: (err: any, product: this, numAffected: number) => void): Promise<this>;
+    save(options?: SaveOptions, fn?: (err: any, product: this) => void): Promise<this>;
```

Comment:
`numAffected` is still there: http://mongoosejs.com/docs/api.html#document_Document-save

Reply:
Their migration doc states that numAffected is no longer passed: https://github.com/Automattic/mongoose/blob/master/migrating_to_5.md#numaffected-and-save

Reply:
Oh you're totally right. This is fine. I'll make a PR to update their API docs.